quarkus/container/Dockerfile

```
@@ -18,11 +18,13 @@ RUN mv /tmp/keycloak/keycloak-* /opt/keycloak && mkdir -p /opt/keycloak/data
 RUN chmod -R g+rwX /opt/keycloak
 
 FROM registry.access.redhat.com/ubi8-minimal
+ENV LANG en_US.UTF-8
 
 COPY --from=build-env --chown=1000:0 /opt/keycloak /opt/keycloak
 
 RUN microdnf update -y && \
-    microdnf install -y java-11-openjdk-headless && microdnf clean all && rm -rf /var/cache/yum/* && \
+    microdnf install -y java-11-openjdk-headless glibc-langpack-en && microdnf clean all && rm -rf /var/cache/yum/* && \
```

Was curious, I see this isn't excluding the docs for these packages today.  Is there anything that would prevent us from doing that?  Checking locally it looks like the size drops by ~2MB when you add the `--nodocs` flag. Which that's not a _huge_ amount by itself, but I imagine we're hopeful many folks are pulling this image and it could add up over time.

```suggestion
    microdnf install -y --nodocs java-11-openjdk-headless glibc-langpack-en && microdnf clean all && rm -rf /var/cache/yum/* && \
```

---

I'd say no, wdyt @pedroigor @stianst @vmuzikar? Should i change the line in this PR as suggested or should we open another issue for this one?

---

@vmuzikar i added the suggestion for now, so perhaps this PR can get merged quickly. Happy to remove again if someone wants, but i also think we can sneak it in here :)